generate-workflow.py

```diff
@@ -150,7 +157,7 @@ class BenchmarkInfo(NamedTuple):
         {cmake_checkedc} -DCMAKE_C_FLAGS="-w" ..
         bear make
```

Comment:
```suggestion
        bear make zlib
```
Let's be consistent (and make the build a little faster).  This will have the additional effect that the compilation database will only include the `zlib` target, so you might be able to remove the `--skip` option, but I'll leave it up to you whether you want to try that now or leave it for later.  (Perhaps in general, if we pick the right target, we should never need a `--skip` option?)

Reply:
I was planning to take a look at all the `--skip` options once we merge the PR for handling multiple definitions of main.